OpenSim/Examples/CMakeLists.txt

```diff
@@ -34,4 +34,9 @@ elseif()
 
 endif()
 
+# We do not install the BuildDynamicWalker example, because it is an assignment
+# in the OpenSim class. The solution to the BuildDynamicWalker assignment is in
```

Comment:
"assignment in the OpenSim class" – I assume you mean ME/BIOE 485, but "assignment" and "class" are ambiguous here. I would replace with something like "…because it is assigned to students as homework."

Reply:
Addressed.